From 50323cee1734eb5ce061fac895ae7e77a3319913 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Wed, 6 Aug 2014 03:45:49 -0400
Subject: [acpi] numa: Make __acpi_map_pxm_to_node(), acpi_get_pxm() static

Message-id: <20140806034548.1438.51266.stgit@gir.stowe>
Patchwork-id: 87425
O-Subject: [RHEL7.1 PATCH 16/69] ACPI / numa: Make __acpi_map_pxm_to_node(), acpi_get_pxm() static
Bugzilla: 1110896
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Dean Nelson <dnelson@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1110896
Upstream Status: d79ed248d923f219053760376a33371894a6d47c

commit d79ed248d923f219053760376a33371894a6d47c
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Jan 24 15:48:01 2014 -0700

    ACPI / numa: Make __acpi_map_pxm_to_node(), acpi_get_pxm() static

    __acpi_map_pxm_to_node() and acpi_get_pxm() are only used within
    drivers/acpi/numa.c.  This makes them static and removes their
    declarations.

    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Acked-by: Rafael J. Wysocki <rafael.j.wysocki@intel.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/acpi/numa.c b/drivers/acpi/numa.c
index 98197cd..7d83ab8 100644
--- a/drivers/acpi/numa.c
+++ b/drivers/acpi/numa.c
@@ -61,7 +61,7 @@ int node_to_pxm(int node)
  return node_to_pxm_map[node];
 }
 
-void __acpi_map_pxm_to_node(int pxm, int node)
+static void __acpi_map_pxm_to_node(int pxm, int node)
 {
  if (pxm_to_node_map[pxm] == NUMA_NO_NODE || node < pxm_to_node_map[pxm])
   pxm_to_node_map[pxm] = node;
@@ -315,7 +315,7 @@ int __init acpi_numa_init(void)
  return 0;
 }
 
-int acpi_get_pxm(acpi_handle h)
+static int acpi_get_pxm(acpi_handle h)
 {
  unsigned long long pxm;
  acpi_status status;
diff --git a/include/acpi/acpi_numa.h b/include/acpi/acpi_numa.h
index 451823c..94a37cd 100644
--- a/include/acpi/acpi_numa.h
+++ b/include/acpi/acpi_numa.h
@@ -13,7 +13,6 @@
 
 extern int pxm_to_node(int);
 extern int node_to_pxm(int);
-extern void __acpi_map_pxm_to_node(int, int);
 extern int acpi_map_pxm_to_node(int);
 extern unsigned char acpi_srat_revision;
 
diff --git a/include/linux/acpi.h b/include/linux/acpi.h
index d1d2483..73fb559 100644
--- a/include/linux/acpi.h
+++ b/include/linux/acpi.h
@@ -238,13 +238,8 @@ extern void acpi_dmi_osi_linux(int enable, const struct dmi_system_id *d);
 extern void acpi_osi_setup(char *str);
 
 #ifdef CONFIG_ACPI_NUMA
-int acpi_get_pxm(acpi_handle handle);
 int acpi_get_node(acpi_handle handle);
 #else
-static inline int acpi_get_pxm(acpi_handle handle)
-{
- return 0;
-}
 static inline int acpi_get_node(acpi_handle handle)
 {
  return 0;
-- 
1.7.1